custom_model_runner/datarobot_drum/drum/common.py

```diff
@@ -130,6 +130,16 @@ class EnvVarNames:
     DRUM_JAVA_XMX = "DRUM_JAVA_XMX"
 
 
+@contextmanager
+def reroute_stdout_to_stderr():
+    keep = sys.stdout
+    sys.stdout = sys.stderr
+    try:
+        yield
+    finally:
+        sys.stdout = keep
```

Comment:
does this redirect stderr to stdout, or just replace stdout with stderr?

Reply:
redirects stdout to stderr